userguide/storage.rst

```diff
@@ -2378,8 +2378,20 @@ options in the replication task dialog.
    | replicate child       |              |                                                                                                              |
    | dataset snapshots     |              |                                                                                                              |
    +-----------------------+--------------+--------------------------------------------------------------------------------------------------------------+
-   | Delete stale          | checkbox     | Set to delete previous snapshots from the remote or destination system                                       |
-   | snapshots             |              | which are no longer present on the source computer.                                                          |
+   | Snapshot retention    | drop-down    | Define when snapshots are deleted from the destination system:                                               |
+   | policy                | menu         |                                                                                                              |
+   |                       |              | * *Same as Source*: duplicate the snapshot lifetime setting from the source system. Removing a snapshot from |
+   |                       |              |   the source system also removes that snapshot from the destination system.                                  |
+   |                       |              | * *Custom*: adds :guilabel:`Snapshot lifetime value` and :guilabel:`Snapshot lifetime unit` fields to define |
+   |                       |              |   a snapshot lifetime for the destination system.                                                            |
+   |                       |              | * *None*: never delete snapshots from the destination system.                                                |
+   |                       |              |                                                                                                              |
+   +-----------------------+--------------+--------------------------------------------------------------------------------------------------------------+
+   | Snapshot lifetime     | integer      | How long a snapshot will remain on the destination system. Enter a number.                                   |
```

Comment:
Roughly: Time before snapshot is deleted on destination system. This field is a number that works in concert with :guilabel:`Snapshot Lifetime Unit`.